Spawn-App-iOS-SwiftUI/Views/Helpers/Constants.swift

```diff
@@ -9,5 +9,5 @@ import SwiftUI
 
 let universalRectangleCornerRadius: CGFloat = 20
 let eventColors: [Color] = [Color(hex: "#8084ac"), Color(hex: "#704444"), Color(hex: "#b0442c"), Color(hex: "#889c6c")]
-let universalBackgroundColor: Color = Color(hex: "#d0ccc4")
+let universalBackgroundColor: Color = Color(hex: "#E7E7DD")
```

Comment:
Side-by-side, on the same screen, these look like different colours, between the figma design, and what you've now got in the app. I initially used a tool to detect the colour (the one on figma is noticeably darker). 

How come you changed it? 

<img width="990" alt="image" src="https://github.com/user-attachments/assets/fbb56a34-4292-4234-8545-06770348ef04">

Reply:
Oh I got the color from the Figma design
<img width="838" alt="Screenshot 2024-11-19 at 5 55 21 PM" src="https://github.com/user-attachments/assets/b8bcce73-9b1b-4ea6-a369-23c33fab1284">

Reply:
@miketham24 I think that the background color, though (e.g. on the feedview page) is different than on the modal popup

Reply:
You are right that inside of Figma it says that for both, though. So, maybe my eyes are just off

Reply:
Side-by-side, they definitely look different (darker in Figma than on the phone simulator):

![image](https://github.com/user-attachments/assets/78899642-7ee3-4f4d-bab9-ed2c57a47aca)